7.0/Dockerfile

```diff
@@ -39,7 +39,7 @@ RUN set -x; \
         && /install/postgres.sh \
         && /install/wkhtml_12_1_2.sh \
         && /install/dev_package.sh \
-        && pip install -U pip setuptools \
+        && pip install -U pip "setuptools<45" \
```

Comment:
&& pip install -U pip setuptools<45 didn't work ?

(just to know)

Reply:
`<` character is used to redirect input, you have to put the quotes to ignore this behavior. If not, you'll get an error like `no such file or directory: 45`

Reply:
thanks a lot for this explanation !